internal/api/arm/error.go

```diff
@@ -60,7 +60,7 @@ func (body *CloudErrorBody) String() string {
 	if len(body.Details) > 0 {
 		details = " Details: "
 		for i, innerErr := range body.Details {
-			details := innerErr.String()
+			details += innerErr.String()
```

Comment:
I'm not 100% sure if this was the original intention and open to changing this, but the original
```go
details := innerErr.String()
```
can't be right